app/views/ServiceConfigPage.scala.html

```diff
@@ -76,23 +72,23 @@ <h2>Columns</h2>
             <ul>
                 @for(env <- ConfigEnvironment.values) {
                     <li>
-                        <input class="column-filter" type="checkbox" id="show-@env.asString" checked="checked" />
+                        <input class="column-filter" type="checkbox" id="show-@env.asString" checked="checked" autocomplete="off" />
```

Comment:
can you add a comment like "without turning autocomplete off, some browsers (e.g. chrome) will set the checkbox state inconsistent with the viewed data when returning to the page"